etl/grapher_model.py

```diff
@@ -103,6 +103,25 @@ class Entity(Base):
     code: Mapped[Optional[str]] = mapped_column(VARCHAR(255))
     updatedAt: Mapped[Optional[datetime]] = mapped_column(DateTime, init=False)
 
+    @classmethod
+    def load_entity_mapping(cls, session: Session, entity_ids: List[int]) -> Dict[int, str]:
```

Comment:
This looks good. (If we find it too slow, we can try `pd.read_sql` instead, but for hundreds of records this should be fast enough).